@xen-orchestra/fs/src/smb-mount.js

```diff
@@ -0,0 +1,18 @@
+import { parse } from 'xo-remote-parser'
+
+import MountHandler from './_mount'
+
+export default class SmbMountHandler extends MountHandler {
+  constructor(remote, opts) {
+    const { domain, host, password, username } = parse(remote.url)
+    super(remote, opts, {
+      type: 'cifs',
+      device: `//${host}`,
+      options: `user=${username},password=${password},domain=${domain}`,
```

Comment:
I think it would be better to pass `username` and `password` via the `USER` and `PASSWD` environment variable to avoid issues with special characters like `,`.